packages/system/package.json

```diff
@@ -9,15 +9,18 @@
   "dependencies": {
     "@emotion/core": "10.1.1",
     "@theme-ui/css": "0.3.3",
+    "emotion": "10.0.27",
```

Comment:
zweimal emotion in dieser Datei. Sollte denke ich nicht so sein.

Reply:
Ja einmal reicht :D